libraries/abstractions/ble_hal/include/bt_hal_manager_adapter_ble.h

```diff
@@ -959,6 +959,19 @@ typedef struct
                                                bool bSetScanRsp,
                                                uint8_t * pucData,
                                                size_t xDataLen );
+
+    /**
+     *
+     * @brief Sets the LE background scan interval and window in units of N*0.625 msec.
+     *
+     * @param[in] ucAdapterIf Adapter interface ID. Returned from BTRegisterBleAdapterCallback_t after calling pxRegisterBleApp.
+     * @param[in] ulScanInterval
+     * @param[in] ulScanWindow
+     * @return Returns eBTStatusSuccess on successful call.
+     */
+    BTStatus_t ( * pxSetBgScanParameters )( uint8_t ucAdapterIf,
+                                            uint32_t ulScanInterval,
```

Comment:
It is preferable for time values to have a suffix that specifies the units, e.g. "ulTimeS", "ulTimeMs", "ulTimeSeconds", etc.  In this case the units are a little unusual - 5/8ms so this won't work.  However, the naming can at least specify that these are time values _and_ the units should also be listed in the doxygen parameter descriptions:

```c
  /**
    .
    .
     *
     * @param[in] ulScanIntervalTime - scan interval time in units of 0.625Ms
     * @param[in] ulScanWindowTime - scan window time in units of 0.625Ms
     .
     .
     .
     */
uint32_t ulScanIntervalTime,
uint32_t ulScanWindowTime
```